app/helpers/proposals_helper.rb

```diff
@@ -32,4 +32,20 @@ def retire_proposals_options
     Proposal::RETIRE_OPTIONS.collect { |option| [ t("proposals.retire_options.#{option}"), option ] }
   end
 
+  def can_create_document?
+    can?(:create, @document) && @proposal.documents.size < Proposal.max_documents_allowed
+  end
+
+  def author_of_proposal?
+    author_of?(@proposal, current_user)
+  end
+
+  def current_editable?
+    current_user && @proposal.editable_by?(current_user)
+  end
+
+  def show_actions_menu?
```

Comment:
I get why you refactored this but thinking again that this is a Helper common to other Proposal stuff, and that `show_actions_menu` is not very specific about (and doesn't seem to be candidate to be reused on other parts... just on that proposal show view) I think leaving that 3 OR's on the view would be better to easily check there when the html will be rendered. It may be a very subjective opinion but for me if its not reusable, or complex logic then there is no need to go on a helper.